Libraries/LibJS/Runtime/Shape.cpp

```diff
@@ -196,7 +196,8 @@ void Shape::reconfigure_property_in_unique_shape(const StringOrSymbol& property_
     ASSERT(is_unique());
     ASSERT(m_property_table);
     ASSERT(m_property_table->contains(property_name));
-    m_property_table->set(property_name, { m_property_table->size(), attributes });
+    auto metadata = m_property_table->get(property_name).value();
+    m_property_table->set(property_name, { metadata.offset, attributes });
```

Comment:
This is two separate hash lookups, let's avoid one :)
```cpp
auto it = m_property_table->find(property_name);
ASSERT(it != m_property_table->end());
it->value = { metadata.offset, attributes };
```

(And with this assertion added, we can drop the `contains()` assertion above)